From 351c63ace66f8d9c5bb9663c66bc851bf600239a Mon Sep 17 00:00:00 2001
From: ZhongWen <b47623@freescale.com>
Date: Tue, 10 Dec 2013 10:10:09 +0800
Subject: [PATCH 045/115] ENGR00288589 Set up WfdSink.apk Chinese language

make some modifis in strings.xml and WfdSinkActivity.java.
add values-en-rUS and values-zh-rCN files to make WfdSink.apk Chinese language.

Signed-off-by: ZhongWen <b47623@freescale.com>
---
 WfdSink/res/values-en-rUS/color.xml                |  6 +++++
 WfdSink/res/values-en-rUS/strings.xml              | 29 ++++++++++++++++++++
 WfdSink/res/values-en-rUS/styles.xml               | 31 ++++++++++++++++++++++
 WfdSink/res/values-en-rUS/themes.xml               | 10 +++++++
 WfdSink/res/values-zh-rCN/color.xml                |  6 +++++
 WfdSink/res/values-zh-rCN/strings.xml              | 29 ++++++++++++++++++++
 WfdSink/res/values-zh-rCN/styles.xml               | 31 ++++++++++++++++++++++
 WfdSink/res/values-zh-rCN/themes.xml               | 10 +++++++
 WfdSink/res/values/strings.xml                     | 15 ++++++++++-
 .../src/com/freescale/wfdsink/WfdSinkActivity.java | 27 ++++++++++---------
 10 files changed, 180 insertions(+), 14 deletions(-)
 create mode 100755 WfdSink/res/values-en-rUS/color.xml
 create mode 100755 WfdSink/res/values-en-rUS/strings.xml
 create mode 100755 WfdSink/res/values-en-rUS/styles.xml
 create mode 100755 WfdSink/res/values-en-rUS/themes.xml
 create mode 100755 WfdSink/res/values-zh-rCN/color.xml
 create mode 100755 WfdSink/res/values-zh-rCN/strings.xml
 create mode 100755 WfdSink/res/values-zh-rCN/styles.xml
 create mode 100755 WfdSink/res/values-zh-rCN/themes.xml

diff --git a/WfdSink/res/values-en-rUS/color.xml b/WfdSink/res/values-en-rUS/color.xml
new file mode 100755
index 0000000..f68349e
--- /dev/null
+++ b/WfdSink/res/values-en-rUS/color.xml
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <color name="qs_text_title">#ffe0e0e0</color>
+
+</resources>
\ No newline at end of file
diff --git a/WfdSink/res/values-en-rUS/strings.xml b/WfdSink/res/values-en-rUS/strings.xml
new file mode 100755
index 0000000..8d2dbbe
--- /dev/null
+++ b/WfdSink/res/values-en-rUS/strings.xml
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name">WfdSink</string>
+    <string name="hello_world">Hello world!</string>
+    <string name="menu_settings">Settings</string>
+    <string name="title_activity_second">SecondActivity</string>
+    <string name="search">Avaiable devices</string>
+    <string name="current">Local device Information</string>
+    <string name="status">status</string>
+    <string name="setting_key">setting_key</string>
+    <string name="setting_summary">setting_summary</string>
+    <string name="setting_title">setting_title</string>	
+    
+    <string name="wifi_no_open">WI-FI has not open !</string>	
+    <string name="wifi_cancel_exit">Are you sure to open now? If you cancel then will exit!</string>
+    <string name="wifi_confirm">Confirm</string>
+    <string name="wifi_cancel">Cancel</string>
+    
+    <string name="program_exit">Program exit ?</string>
+    <string name="program_exit_sure">Are you sure to exit the program ?</string>
+    
+    <string name="wifi_ssid">SSID</string>
+    <string name="wifi_status_connected">Status:Connected</string>
+    <string name="wifi_status_searching">Status:Searching for peers</string>
+    <string name="wifi_status_disconnected">Status:Disconnected!</string>
+    
+    
+</resources>
diff --git a/WfdSink/res/values-en-rUS/styles.xml b/WfdSink/res/values-en-rUS/styles.xml
new file mode 100755
index 0000000..93c6707
--- /dev/null
+++ b/WfdSink/res/values-en-rUS/styles.xml
@@ -0,0 +1,31 @@
+<resources>
+
+    <!--
+        Base application theme, dependent on API level. This theme is replaced
+        by AppBaseTheme from res/values-vXX/styles.xml on newer devices.
+    -->
+    <style name="AppBaseTheme" parent="android:Theme.Light">
+        <!--
+            Theme customizations available in newer API levels can go in
+            res/values-vXX/styles.xml, while customizations related to
+            backward-compatibility can go here.
+        -->
+    </style>
+
+    <!-- Application theme. -->
+    <style name="AppTheme" parent="@android:style/Theme.Black">
+        <!-- All customizations that are NOT specific to a particular API-level can go here. -->
+    </style>
+	
+    <style name="style_quicksettings_category_text">
+        <item name="android:textColor"> @color/qs_text_title</item>
+        <item name="android:textSize">20sp</item>
+        <item name="android:fontFamily">sans-serif</item>
+    </style>
+    
+     <style name="style_quicksettings_category_title">
+        <item name="android:background"> @drawable/bg_title</item>
+        <item name="android:padding">4dp</item>
+    </style>	
+
+</resources>
diff --git a/WfdSink/res/values-en-rUS/themes.xml b/WfdSink/res/values-en-rUS/themes.xml
new file mode 100755
index 0000000..3efaeb9
--- /dev/null
+++ b/WfdSink/res/values-en-rUS/themes.xml
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="theme">
+        <item name="android:windowNoTitle">true</item>
+        <item name="android:windowFullscreen">?android:windowNoTitle</item>
+        <item name="android:windowBackground">@android:drawable/screen_background_dark</item>
+        <item name="android:windowFrame">@null</item>
+        <item name="android:windowAnimationStyle">@android:style/Animation.Activity</item>
+    </style>
+</resources>
diff --git a/WfdSink/res/values-zh-rCN/color.xml b/WfdSink/res/values-zh-rCN/color.xml
new file mode 100755
index 0000000..f68349e
--- /dev/null
+++ b/WfdSink/res/values-zh-rCN/color.xml
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <color name="qs_text_title">#ffe0e0e0</color>
+
+</resources>
\ No newline at end of file
diff --git a/WfdSink/res/values-zh-rCN/strings.xml b/WfdSink/res/values-zh-rCN/strings.xml
new file mode 100755
index 0000000..ec608e6
--- /dev/null
+++ b/WfdSink/res/values-zh-rCN/strings.xml
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name">无线显示</string>
+    <string name="hello_world">Hello world!</string>
+    <string name="menu_settings">设置</string>
+    <string name="title_activity_second">SecondActivity</string>
+    <string name="search">Avaiable devices</string>
+    <string name="current">Local device Information</string>
+    <string name="status">状态</string>
+    <string name="setting_key">设置按钮</string>
+    <string name="setting_summary">setting_summary</string>
+    <string name="setting_title">setting_title</string>	
+    
+    <string name="wifi_no_open">WI-FI 没有打开 !</string>	
+    <string name="wifi_cancel_exit">确定现在开启WI-FI吗? 如果取消则退出!</string>
+    <string name="wifi_confirm">确定</string>
+    <string name="wifi_cancel">取消</string>
+    
+    <string name="program_exit">程序退出 ?</string>
+    <string name="program_exit_sure">确定要退出程序吗 ?</string>
+    
+    <string name="wifi_ssid">设备</string>
+    <string name="wifi_status_connected">状态:已连接</string>
+    <string name="wifi_status_searching">状态:正在寻找其他设备</string>
+    <string name="wifi_status_disconnected">状态:已断开</string>
+    
+    
+</resources>
diff --git a/WfdSink/res/values-zh-rCN/styles.xml b/WfdSink/res/values-zh-rCN/styles.xml
new file mode 100755
index 0000000..93c6707
--- /dev/null
+++ b/WfdSink/res/values-zh-rCN/styles.xml
@@ -0,0 +1,31 @@
+<resources>
+
+    <!--
+        Base application theme, dependent on API level. This theme is replaced
+        by AppBaseTheme from res/values-vXX/styles.xml on newer devices.
+    -->
+    <style name="AppBaseTheme" parent="android:Theme.Light">
+        <!--
+            Theme customizations available in newer API levels can go in
+            res/values-vXX/styles.xml, while customizations related to
+            backward-compatibility can go here.
+        -->
+    </style>
+
+    <!-- Application theme. -->
+    <style name="AppTheme" parent="@android:style/Theme.Black">
+        <!-- All customizations that are NOT specific to a particular API-level can go here. -->
+    </style>
+	
+    <style name="style_quicksettings_category_text">
+        <item name="android:textColor"> @color/qs_text_title</item>
+        <item name="android:textSize">20sp</item>
+        <item name="android:fontFamily">sans-serif</item>
+    </style>
+    
+     <style name="style_quicksettings_category_title">
+        <item name="android:background"> @drawable/bg_title</item>
+        <item name="android:padding">4dp</item>
+    </style>	
+
+</resources>
diff --git a/WfdSink/res/values-zh-rCN/themes.xml b/WfdSink/res/values-zh-rCN/themes.xml
new file mode 100755
index 0000000..3efaeb9
--- /dev/null
+++ b/WfdSink/res/values-zh-rCN/themes.xml
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="theme">
+        <item name="android:windowNoTitle">true</item>
+        <item name="android:windowFullscreen">?android:windowNoTitle</item>
+        <item name="android:windowBackground">@android:drawable/screen_background_dark</item>
+        <item name="android:windowFrame">@null</item>
+        <item name="android:windowAnimationStyle">@android:style/Animation.Activity</item>
+    </style>
+</resources>
diff --git a/WfdSink/res/values/strings.xml b/WfdSink/res/values/strings.xml
index eee7812..484a269 100755
--- a/WfdSink/res/values/strings.xml
+++ b/WfdSink/res/values/strings.xml
@@ -10,5 +10,18 @@
     <string name="status">status</string>
     <string name="setting_key">setting_key</string>
     <string name="setting_summary">setting_summary</string>
-    <string name="setting_title">setting_title</string>	
+    <string name="setting_title">setting_title</string>
+    <string name="wifi_no_open">WI-FI has not open !</string>
+    <string name="wifi_cancel_exit">Are you sure to open now? If you cancel then will exit!</string>
+    <string name="wifi_confirm">Confirm</string>
+    <string name="wifi_cancel">Cancel</string>
+
+    <string name="program_exit">Program exit ?</string>
+    <string name="program_exit_sure">Are you sure to exit the program ?</string>
+
+    <string name="wifi_ssid">SSID:</string>
+    <string name="wifi_status_connected">Status:Connected</string>
+    <string name="wifi_status_searching">Status:Searching for peers</string>
+    <string name="wifi_status_disconnected">Status:Disconnected!</string>
+	
 </resources>
diff --git a/WfdSink/src/com/freescale/wfdsink/WfdSinkActivity.java b/WfdSink/src/com/freescale/wfdsink/WfdSinkActivity.java
index 52ab90d..bc38ead 100755
--- a/WfdSink/src/com/freescale/wfdsink/WfdSinkActivity.java
+++ b/WfdSink/src/com/freescale/wfdsink/WfdSinkActivity.java
@@ -147,7 +147,8 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
                 currentdevice = (TextView)findViewById(R.id.currentdevice);
                 if (mThisName != mWfdSink.getDeviceName())
                     mThisName = mWfdSink.getDeviceName();
-                currentdevice.setText("SSID:" + mWfdSink.getDeviceName());
+                String ssid = getString(R.string.wifi_ssid) + ":";
+                currentdevice.setText(ssid + mWfdSink.getDeviceName());
             }
 
         }
@@ -239,8 +240,8 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
 
     private void wifiDialog() {
         Dialog dialog = new AlertDialog.Builder(WfdSinkActivity.this)
-            .setTitle("WI-FI hasn't open ! ").setMessage("Are you sure to open now? If you cancel then will exit!").setIcon(R.drawable.ic_hdmi)
-            .setPositiveButton("confirm", new DialogInterface.OnClickListener() {
+            .setTitle(R.string.wifi_no_open).setMessage(R.string.wifi_cancel_exit).setIcon(R.drawable.ic_hdmi)
+            .setPositiveButton(R.string.wifi_confirm, new DialogInterface.OnClickListener() {
 
                     @Override
                     public void onClick(DialogInterface dialog, int which) {
@@ -248,7 +249,7 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
                     startActivity(new Intent(Settings.ACTION_WIFI_SETTINGS));
 
                     }
-                    }).setNegativeButton("cancel", new DialogInterface.OnClickListener() {
+                    }).setNegativeButton(R.string.wifi_cancel, new DialogInterface.OnClickListener() {
 
                         @Override
                         public void onClick(DialogInterface dialog, int which) {
@@ -260,10 +261,10 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
 
     private void exitDialog() {
         Dialog dialog = new AlertDialog.Builder(WfdSinkActivity.this)
-            .setTitle("Program exit ?")
-            .setMessage("Are you sure to exit the program ?")
+            .setTitle(R.string.program_exit)
+            .setMessage(R.string.program_exit_sure)
             .setIcon(R.drawable.ic_hdmi)
-            .setPositiveButton("confirm", new DialogInterface.OnClickListener(){
+            .setPositiveButton(R.string.wifi_confirm, new DialogInterface.OnClickListener(){
 
                     @Override
                     public void onClick(DialogInterface arg0, int arg1) {
@@ -271,7 +272,7 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
 
                     }
 
-                    }).setNegativeButton("cancel", new DialogInterface.OnClickListener() {
+                    }).setNegativeButton(R.string.wifi_cancel, new DialogInterface.OnClickListener() {
 
                         @Override
                         public void onClick(DialogInterface dialog, int which) {
@@ -295,7 +296,7 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
                     handleUpdateSurface(holder);
                     break;
 
-                case START_PLAY:                	
+                case START_PLAY:
                     handleStartPlay();
                     break;
 
@@ -304,7 +305,7 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
                     break;
 
                 case DO_CONNECTED:
-                    status.setText("Status:Connected");
+                    status.setText(R.string.wifi_status_connected);
                     boolean connected = (msg.arg1 == 1);
                     handleConnected(connected);
                     sink_main.setSystemUiVisibility(View.SYSTEM_UI_FLAG_HIDE_NAVIGATION);
@@ -376,7 +377,7 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
         mConnected = connected;
         if (mConnected) {
             stopSearch();
-        	layout.setVisibility(View.INVISIBLE);
+            layout.setVisibility(View.INVISIBLE);
             mSurfaceView.setVisibility(View.VISIBLE);
             mGridView.setVisibility(View.GONE);
             startPlayer();
@@ -388,7 +389,7 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
     }
 
     private void startSearch() {
-        status.setText("Status:Searching for peers");
+        status.setText(R.string.wifi_status_searching);
         mWfdSink.startSearch();
         //mWfdSink.setDeviceName("Android_me");
     }
@@ -407,7 +408,7 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
     }
 
     private void stopPlayer() {
-        status.setText("Status:Disconnected!");
+        status.setText(R.string.wifi_status_disconnected);
         Message msg = mHandler.obtainMessage(STOP_PLAY);
         mHandler.sendMessage(msg);
     }
-- 
1.8.0

